installers/metrics/ansible/roles/pgo-metrics/files/prometheus.yml

```diff
@@ -0,0 +1,63 @@
+---
+global:
+  scrape_interval: 30s
+  scrape_timeout: 30s
+  evaluation_interval: 5s
+
+scrape_configs:
+- job_name: crunchy-collect
+  kubernetes_sd_configs:
+  - role: pod
+
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_pod_label_crunchy_collect]
+    action: keep
+    regex: true
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 5432
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 10000
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 8009
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 2022
+  - source_labels: [__meta_kubernetes_namespace]
+    action: replace
+    target_label: kubernetes_namespace
+  - source_labels: [__meta_kubernetes_pod_name]
+    target_label: pod
+  - source_labels: [__meta_kubernetes_pod_name]
+    regex: (^[^-]*).*
+    target_label: instance
+    replacement: '$1'
+  - source_labels: [__meta_kubernetes_namespace,instance]
+    target_label: pg_cluster
+    separator: ': '
+    replacement: '$1$2'
+  - source_labels: [__meta_kubernetes_pod_ip]
+    target_label: ip
+    replacement: '$1'
+  - source_labels: [__meta_kubernetes_pod_label_deployment_name]
+    target_label: deployment
+    replacement: '$1'
+  - source_labels: [__meta_kubernetes_pod_label_service_name]
+    target_label: service
+    replacement: '$1'
+  - source_labels: [__meta_kubernetes_pod_label_role]
+    target_label: role
+    replacement: '$1'
+  - source_labels: [dbname]
+    target_label: dbname
+    replacement: '$1'
+  - source_labels: [relname]
+    target_label: relname
+    replacement: '$1'
+  - source_labels: [schemaname]
+    target_label: schemaname
+    replacement: '$1'
```

Comment:
These three look like no-ops. Perhaps I'm misunderstanding?

Reply:
See comment above regarding the containers `prometheus.yaml` config that is now part of pgmonitor (where we will be grabbing it from moving forward).